campaign/sample/lore.txt

```diff
@@ -1,17 +1,435 @@
+
+- Haven was gone in a few days, what was left of the Union weeks later with barely a warning. However, the Empire had more time to prepare, as most of its territory would not be affected for years. They started evacuating as many people as possible towards the furthest reaches that would be affected last. However, only a fraction of the total population could be evacuated in time on most worlds.
+Their preliminary research led them to conclude that the Event would spend itself after covering the entire galaxy, its satellites, and go much of the way towards the closest neighbors in the Local Group before dissipating. Trying to outrun it would have been futile.
+
+- The Empire had inherited some knowledge of the Eternity Gate and the Mandate's work on it, and had done some research on it, but otherwise kept it quiet. However, with the Event closing in on them, and the Gate's structure apparently slowing its progression down, they considered it for escape. They didn't know much about what was on the other side, but knew it was inhabitable in some form and had enough data on the general conditions to put emergency settlement kits to hopefully survive there.
+So they turned the closest planets into vast temporary shelters to house the refugees before evacuation, while reactivating the still mostly functional Mandate relay systems and building fleets of colony ships capable of following them and crossing the rift at its end.
+It is unclear how many ships and refugees took of before the event swept those last worlds, or how many made it all the way through the Gate.
+
+
+Silence Era
+
+- After the Event, some of the few survivors in deep hibernation started to wake up in an emptied galaxy. Most infrastructure and automated systems were still running, and while some areas still had active military bots and other automated hazards, most of the galaxy was mostly safe.
+Some of the survivors managed to get interstellar communication and starships working, contacting each-other and searching worlds for other survivors. They soon discovered the final evacuation plans of the Empire towards the Eternity Gate. They decided to follow them, as many evacuation ships were still present and functional, and they judged they were too few to really restart civilization in this galaxy.
+Knowing that other survivors may wake up after themselves, as there were little chances that they had found everyone, they prepared things for others trying to follow them afterwards.
+They left message beacons running, put supplies together in many places, deactivated some dangerous automated systems, set up warp gates for long-term autonomous function and self-repair, left control codes they could find for still active military automata�
```

Comment:
This sentence also has some weird character at the end, is it a broken period or was the sentence cut off?
![image](https://github.com/ZeroK-RTS/Chobby/assets/2573076/78768582-8183-4fd7-b77c-f5f04c226854)

Reply:
Fixed

Reply:
Fixed